mf-dev/dme/dme-import-map.json

```diff
@@ -16,7 +16,7 @@
     "@digital-enabler/dme-datasources-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-datasources-list@1.0.0/app.js",
     "@digital-enabler/dme-datasource-details": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-datasource-details@1.0.2/app.js",
     "@digital-enabler/dme-flows-list": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flows-list@1.0.0/app.js",
-    "@digital-enabler/dme-flow-editor": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flow-editor@1.1.2/app.js",
+    "@digital-enabler/dme-flow-editor": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flow-editor@1.2.3/app.js",
```

Comment:
1.1.3 is for the validation bug fix and then i added another commit for this feature so its 1.2.3, should i create a separate PR for the 1.2.3?

Reply:
Please, change it to 1.1.3. the 1.2.0 is not yet approved

Reply:
1.1.3 was a bug fix for the validation not working, instead of 1.2.3 it should be 1.1.4 for the [Flow target no outputs](https://github.com/digital-enabler/demf-flow-editor/pull/42)

Reply:
```suggestion
    "@digital-enabler/dme-flow-editor": "https://cdn.jsdelivr.net/npm/@digital-enabler/dme-flow-editor@1.2.0/app.js",
```